python/cross_service/resilient_service/auto_scaler.py

```diff
@@ -50,43 +49,120 @@ def __init__(
         self.ec2_client = ec2_client
         self.ssm_client = ssm_client
         self.iam_client = iam_client
-        self.launch_template_name = f"{resource_prefix}-template"
-        self.group_name = f"{resource_prefix}-group"
+        sts_client = boto3.client("sts")
+        self.account_id = sts_client.get_caller_identity()["Account"]
+
+        rand_int = random.randint(100, 999)
```

Comment:
If we are adding a random component to the names of the resources, what happens when the user tries to run only the -destroy option of the scenario?